software/wmla120.py

```diff
@@ -1685,10 +1688,10 @@ def init_clients(self):
                 if key not in validation_status:
                     validation_status[key] = f'{self.v_status}'
             print("\n   *** Validation Status ***\n")
-            for key,val in validation_status.items():
+            for key, val in validation_status.items():
                 print(f'{key} = {val}')
 
-            print('\nVerification Completed\n')
+            print('\nVerfication Completed\n')
```

Comment:
Typo fixed in PR #517 re-introduced here.

Reply:
fixed